PG/README.md

```diff
@@ -5,10 +5,10 @@ Node version 10.9.0 was used for this SDK.
 
```

Comment:
```js
const PaymentGateway = require('cashfree-sdk').PG;

// Instantiate Cashfree Payment Gateway
const PG = new PaymentGateway({
    env: '<ENVIRONMENT>',
    apiVersion: '<API_VERSION>',
    appId: '<YOUR_APP_ID>',
    secretKey: '<YOUR_SECRET_KEY>',
});
```

@premaddx Here, `PG` variable below where does it point to ... `PaymentGateway` or `PG` variable above? 

```js
PG.Orders.GetLink({
    orderId: '<ORDER_ID>',
}).then(data => console.log(data)).catch(error => console.error(error));
```

Reply:
PG variable above.

Reply:
@premaddx 

```js
import React, {Component} from 'react';

// Here, we should not be creating the variable with the same name as the imported one right?
const Component = new SomeClass({ ... })

// This is confusing
Component.someMethod()
```